src/styles/main.scss

```
@@ -1,7 +1,160 @@
-@import "utils";
-@import "fonts";
-@import "typography";
+@mixin desktop {
+  @media (min-width: 1024px) {
+    @content;
+  }
+}
+
+.page {
+  font-family: 'Inter', sans-serif;
+
+  &__body {
+    margin: 0;
+  }
+}
+
+.container {
+  margin: 0 auto;
+  max-width: 1020px;
+  padding: 0 20px;
+
+  @include desktop {
+    padding: 0 120px;
+  }
+}
+
+.top-actions {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
```

```suggestion
    display: flex;
    justify-content: space-between;
    align-items: center;
```
can make mixin for this "trinity"

---

thanks